senders/pagerduty/send.go

```diff
@@ -61,10 +62,16 @@ func (sender *Sender) buildEvent(events moira.NotificationEvents, contact moira.
 		Details:   details,
 	}
 
+	lAction := "trigger"
```

Comment:
What is the meaning of name `lAction`? I think that just `action` will be better.

Reply:
l is just prefix, to simvolise that this variable is local. This name is passed throw golang lint, and `make lint` from moira project we can't launch due it is fail in `make lint`

```
goroutine 9168 [semacquire]:
sync.runtime_SemacquireMutex(0xc02acff93c, 0xc001902a00, 0x1)
        /usr/lib/go-1.13/src/runtime/sema.go:71 +0x47
sync.(*Mutex).lockSlow(0xc02acff938)
        /usr/lib/go-1.13/src/sync/mutex.go:138 +0xfc
sync.(*Mutex).Lock(...)
        /usr/lib/go-1.13/src/sync/mutex.go:81
github.com/golangci/go-tools/ssa.(*Program).needMethodsOf(0xc02acff900, 0x1048d00, 0xc0a3de7830)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/methods.go:144 +0xa1
github.com/golangci/go-tools/ssa.emitConv(0xc04d0fd400, 0x105bfe0, 0xc0a3e21560, 0x1048c80, 0xc00191e240, 0x105d660, 0xc0a3e215c0)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/emit.go:210 +0x336
github.com/golangci/go-tools/ssa.(*builder).emitCallArgs(0xc094aefc80, 0xc04d0fd400, 0xc01eb59200, 0xc01eae7380, 0x0, 0x0, 0x4, 0x40e8c8, 0x80, 0xd94a80)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:922 +0x5e0
github.com/golangci/go-tools/ssa.(*builder).setCall(0xc094aefc80, 0xc04d0fd400, 0xc01eae7380, 0xc0a3e364c0)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:969 +0x102
github.com/golangci/go-tools/ssa.(*builder).expr0(0xc094aefc80, 0xc04d0fd400, 0x10510e0, 0xc01eae7380, 0x7, 0x1048b40, 0x16b9d80, 0x0, 0x0, 0x0, ...)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:596 +0x29b6
github.com/golangci/go-tools/ssa.(*builder).expr(0xc094aefc80, 0xc04d0fd400, 0x10510e0, 0xc01eae7380, 0x88, 0xe2fe00)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:530 +0x27b
github.com/golangci/go-tools/ssa.(*builder).cond(0xc094aefc80, 0xc04d0fd400, 0x10510e0, 0xc01eae7380, 0xc0a3e384d0, 0xc0a3e38420)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:114 +0x8c
github.com/golangci/go-tools/ssa.(*builder).cond(0xc094aefc80, 0xc04d0fd400, 0x1051a60, 0xc01eae9f60, 0xc0a3e38420, 0xc0a3e384d0)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:103 +0x116
github.com/golangci/go-tools/ssa.(*builder).stmt(0xc094aefc80, 0xc04d0fd400, 0x1051520, 0xc01eae7400)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2114 +0x2709
github.com/golangci/go-tools/ssa.(*builder).stmtList(0xc094aefc80, 0xc04d0fd400, 0xc01eae74c0, 0x3, 0x4)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:790 +0x72
github.com/golangci/go-tools/ssa.(*builder).stmt(0xc094aefc80, 0xc04d0fd400, 0x1051060, 0xc01eaea390)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2102 +0x22a3
github.com/golangci/go-tools/ssa.(*builder).rangeStmt(0xc094aefc80, 0xc04d0fd400, 0xc01eadf7c0, 0x0)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:1949 +0x435
github.com/golangci/go-tools/ssa.(*builder).stmt(0xc094aefc80, 0xc04d0fd400, 0x1051760, 0xc01eadf7c0)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2140 +0x16a2
github.com/golangci/go-tools/ssa.(*builder).stmtList(0xc094aefc80, 0xc04d0fd400, 0xc01eaec140, 0x2, 0x2)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:790 +0x72
github.com/golangci/go-tools/ssa.(*builder).stmt(0xc094aefc80, 0xc04d0fd400, 0x1051060, 0xc01eaea3c0)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2102 +0x22a3
github.com/golangci/go-tools/ssa.(*builder).stmt(0xc094aefc80, 0xc04d0fd400, 0x1051520, 0xc01eae7500)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2116 +0x276f
github.com/golangci/go-tools/ssa.(*builder).stmtList(0xc094aefc80, 0xc04d0fd400, 0xc01e508e00, 0x5, 0x8)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:790 +0x72
github.com/golangci/go-tools/ssa.(*builder).stmt(0xc094aefc80, 0xc04d0fd400, 0x1051060, 0xc01eaea600)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2102 +0x22a3
github.com/golangci/go-tools/ssa.(*builder).buildFunction(0xc094aefc80, 0xc04d0fd400)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2195 +0x2e1
github.com/golangci/go-tools/ssa.(*builder).buildFuncDecl(0xc094aefc80, 0xc04c0b5a40, 0xc01eaea630)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2225 +0xec
github.com/golangci/go-tools/ssa.(*Package).build(0xc04c0b5a40)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2341 +0x884
sync.(*Once).doSlow(0xc04c0b5a6c, 0xc04d55c790)
        /usr/lib/go-1.13/src/sync/once.go:66 +0xe3
sync.(*Once).Do(...)
        /usr/lib/go-1.13/src/sync/once.go:57
github.com/golangci/go-tools/ssa.(*Package).Build(0xc04c0b5a40)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2260 +0x65
github.com/golangci/go-tools/ssa.(*Program).Build.func1(0xc04be88590, 0xc04c0b5a40)
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2244 +0x2b
created by github.com/golangci/go-tools/ssa.(*Program).Build
        /home/vagrant/lib/Golang/pkg/mod/github.com/golangci/go-tools@v0.0.0-20190318055746-e32c54105b7c/ssa/builder.go:2243 +0x118
Makefile:27: recipe for target 'lint' failed
```

Reply:
The point of my previous comment was the following: I do not see the reason for using the prefix for this variable and even more we do not use prefixes in Moira source code. Right now the name for variable is correct.

Reply:
I did not see the issue with make lint as in your traceback before